aptos-move/aptos-gas/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "aptos-gas"
+version = "0.1.0"
+authors = ["Aptos Labs <opensource@aptoslabs.com>"]
+description = "Gas metering for the Aptos blockchain"
+repository = "https://github.com/aptos-labs/aptos-core"
+homepage = "https://aptoslabs.com"
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[dependencies]
+move-binary-format = { git = "https://github.com/move-language/move", rev = "7d28bfa484ef7d9ff225a748832de1cf1b67d77b" }
```

Comment:
Why not use move-deps so the move dependency can be updated in the same place?

Reply:
`move-deps` force feature unification, which is causing some problems.

Reply:
It does make it easier to upgrade Move though. Having more than one versions in the same repo seems a bit inconsistent to me and can potentially be dangerous if there are incompatibilities?

Reply:
> Having more than one versions in the same repo also seems a bit inconsistent to me and can potentially be dangerous if there are incompatibilities?

I think this is debatable. There are legitimate use cases to have different features enabled for different parts of the project. For example, developer tools like CLIs will want to include some optional testing features while the prod VM likely won't, due to security reasons.

Now you can argue that things like these don't necessarily need to be gated behind feature flags and I would agree with you, but unfortunately this is not how things work in the Move repo right now, and it would take us some time to talk to the community and make changes.

Reply:
That makes sense. Could you share what features you're trying to prevent from leaking in here? It seems like this might just be a move fast hack that we could iterate on later so that we translate aptos feature flags into move feature flags?

Reply:
> Could you share what features you're trying to prevent from leaking in here?
If move-cli is present in the build tree, then move-stdlib's "testing" feature will be enabled, even if you don't specify it.

> It seems like this might just be a move fast hack that we could iterate on later so that we translate aptos feature flags into move feature flags?
The problem is in the Move repo and I need to think about it more.